resource_activity/reports/sale_order_report.xml

```diff
@@ -50,12 +50,47 @@
                 <div name="langs" t-if="doc.langs" class="col-xs-3">
                     <strong>Langs:</strong>
                     <t t-foreach="doc.langs" t-as="lang">
-                        <t t-if="lang">                 
+                        <t t-if="lang">
                             <div t-field="lang.name"/>
-                        </t>                                  
+                        </t>
+                    </t>
+                </div>
+
+                <div t-if="doc.location_id.line_ids">
+                    <t t-foreach="doc.location_id.line_ids" t-as="line">
+                        <t t-if="(line.location_id == doc.location_id) and (line.activity_type_id == doc.activity_type)">
+                            <div name="resources" t-if="(doc.activity_id.booked_resources) and (line.bike_number_display)" class="col-xs-3">
+                                <strong>Resources booked:</strong>
+                                <div name="booked_resources_list"
+                                     t-if="(line.bike_number_display == 'list')">
+                                    <t t-foreach="doc.booked_resources" t-as="booked_resource">
+                                        <div t-field="booked_resource.name"/>
+                                    </t>
+                                </div>
+
+                                <div name="booked_resources_category"
+                                     t-if="(line.bike_number_display == 'type')">
+                                    <t t-foreach="doc.get_category_quantity()" t-as="category">
+                                        <t t-esc="category_value"/> <t t-esc="category"/><br/>
+                                    </t>
+                                </div>
+                            </div>
+                        </t>
                     </t>
                 </div>
              </xpath>
+
+
+            <xpath expr="//p[@id='fiscal_position_remark']" position="after">
+                <t t-if="doc.location_id.line_ids">
+                    <t t-foreach="doc.location_id.line_ids" t-as="line">
+                        <t t-if="(line.location_id == doc.location_id) and (line.activity_type_id == doc.activity_type)">
+                            <p style="page-break-before:always;"/>

```

Comment:
TIL

Reply:
[sauce](https://stackoverflow.com/a/32761205)

Reply:
you mean _source_?